odoo/service/server.py

```diff
@@ -332,6 +333,18 @@ def close_socket(self, sock):
                 raise
         sock.close()
 
+    def atexit(self, func):
```

Comment:
If `func` must be called "when the server stops", shouldn't we call this `on_stop`?
`atexit` feels a bit abusive to me, since stopping the server does not necessarily imply exiting (the program)...

Reply:
The name was coined from the standard module atexit, on_stop is better

Reply:
This is for the same reason that our ORM uses awful method names like "unlink" :wink: